.github/actions/pipeline_trigger@v1/dist/index.js

```diff
@@ -0,0 +1,13640 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
This file looks generally mental.  Is it meant to be checked in?

Reply:
Yes, although I agree it's not ideal. From PR description:

> - This action is manually bundled by `@vercel/ncc` to include the dependencies inline. This is necessary because Github actions cannot pull repos from npm (apart from the official Github action packages, I believe), and the alternative was adding a `yarn install && yarn build` to the action which would make it take several minutes.